azurerm/resource_arm_function_app.go

```diff
@@ -37,6 +37,12 @@ func resourceArmFunctionApp() *schema.Resource {
 
 			"location": locationSchema(),
 
+			"kind": {
+				Type:     schema.TypeString,
+				Optional: true,
```

Comment:
I don't see you using the property anywhere, just reading it into state. Is that an oversight or should just maybe just be `Optional`?

Reply:
It is also not set in any of your test configs or documented

Reply:
We can just mark it as `optional` (i believe that is what i wanted to put in my above comment 😅), document it and then continue to use it in the acc test

Reply:
Ahhh, okay I thought you meant the `Optional` in the schema. But you mean marking it as `Optional` in the documentation too 😅? Sure I'll get to it later :) Thanks a lot for the help @katbyte

Reply:
Hi again @katbyte, hope you're good!

I'm scratching my head around the `kind` attribute. Since changing this attribute has no affect and will always be computed, wouldn't it confuse the user to see this value in the schema?

I currently see two possible solutions for it.
1. Use `Optional: true` and `Computed: true` and then specify in the docs that this is computed and changing it will have no effect.
2. Use `UnsafeSetFieldRaw` to be able to set the attribute to the schema in `resourceArmFunctionAppRead` so that we can confirm the `kind` in our acceptance tests.

What do you think?